website/source/docs/configuration/client.html.md

```diff
@@ -28,6 +28,10 @@ client {
 }
 ```
 
+## Plugin Options
+
+Nomad 0.9 now supports pluggable drivers. Operators should use the new [plugin][plugin-stanza] syntax to modify driver configuration. To find the plugin options supported by each individual Nomad driver, please see the [drivers documentation](/docs/drivers/index.html).
```

Comment:
Maybe add a line like this since we expect people to upgrade to 0.9 slowly (and therefore have lots of docs visitors who care about pre-0.9 things):
> The pre-0.9 `client.options` stanza will be supported in 0.9 for backward compatibility but will be removed in a future release.